lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js

```diff
@@ -152,7 +152,7 @@ module.exports = [
         score: 0,
         extendedInfo: {
           value: {
-            0: {value: '6,025'},
+            0: {value: '6,037'},
```

Comment:
TIL that `document.querySelector('html, html /deep/ *')` returns different results for shadow dom v0 vs shadow dom v1! I've used it in the past for v0 (e.g. chromestatus returns correct results), but the dbw tester uses shadow dom v1, so we weren't catching all the nodes that we should have been. No idea why v0 and v1 are different, but they deprecated `/deep/` before v1 came out. So may be it was buggy with the new API and they never fixed it.